src/index.js

```diff
@@ -296,4 +296,17 @@ export default class SecureLS {
     return this.get(this.utils.metaKey, true);
   };
 
+  // enable compatiblity with the sessionStorage/localStorage api
+  getItem(key, isAllKeysData) {
```

Comment:
These methods are not required because we need users to use our API so that we can get.set.remove from our own bucket of information. Using method: `getAllKeys` - will return all the keys so far set. But if we use the methods you added, we won't be able to log those keys. So, please remove them.
Let me know if you need more explanation here.

Reply:
I don't see how these additional methods inhibit getting the information "from our own bucket of information" in any way, they are just references to the original get, set and remove methods.  `getAllKeys` works fine, I locally modified the test in `standard.spec.jsL60` to make sure. You can see for yourself if you like 🙂. 
Also having these functions is essential for our team's project requirements. This way you can use SecureLS as a drop in solution for sessionStorage/localStorage in all kinds of other frameworks and libraries. 
For example, with an apollographql plugin (https://github.com/apollographql/apollo-cache-persist):
`import { InMemoryCache } from 'apollo-cache-inmemory';
import { persistCache } from 'apollo-cache-persist';
import SecureLS from 'secure-ls';

const cache = new InMemoryCache({...});

persistCache({
  cache,
  storage: new SecureLS(),
});
...